src/Digdir.Domain.Dialogporten.Domain/Dialogs/Entities/Activities/DialogActivityType.cs

```diff
@@ -10,36 +10,38 @@ public class DialogActivityType : AbstractLookupEntity<DialogActivityType, Dialo
     public enum Values
     {
         /// <summary>
-        /// Refererer en innsending utført av party som er mottatt hos tjenestetilbyder.
+        /// Refers to a dialog that has been created.
         /// </summary>
-        Submission = 1,
+        DialogCreated = 1,
 
         /// <summary>
-        /// Indikerer en tilbakemelding fra tjenestetilbyder på en innsending. Inneholder
-        /// referanse til den aktuelle innsendingen.
+        /// Refers to a dialog that has been completed.
         /// </summary>
-        Feedback = 2,
+        DialogCompleted = 2,
 
         /// <summary>
-        /// Informasjon fra tjenestetilbyder, ikke (direkte) relatert til noen innsending.
+        /// Refers to a dialog that has been closed.
         /// </summary>
-        Information = 3,
+        DialogClosed = 3,
```

Comment:
Just noticed there are 7 values in this enum, spec. has 6.
`DialogClosed` is not in #858, I'm guessing 'closed' is covered by `DialogCompleted`, @elsand ?

Reply:
Yeah I just saw it in the gist https://gist.github.com/elsand/ef205ad479b3377aea98506808d9a84f

Might be the same enum?

Reply:
This is an oversight; having both is redundant. Suggest we keep DialogClosed only.